### PR TITLE
Ollie Portfolio

### PR DESCRIPTION
**Pull request by olyolyoly**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <link href="css/normalize.css" rel="stylesheet" type="text/css">
+    <link href="css/portfolio.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Amatic+SC" rel="stylesheet">
+    <title>Ollie Portfolio</title>
+</head>
+
+<body>
+    <nav>
+        <div class="menu">
+            <div class="buttons"> HOME </div>
```
Comment (alicemcneil) [inline index.html:15]: could you use` <a>` (anchor/link tags) to make these clickable eventually?

File: css/portfolio.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+.image-title {
+    text-align: center;
+    margin-top: 250px;
+}
+
+.title-style {
+    font-weight: bold;
+    color: white;
+    font-size: 5rem;
+}
```
Comment (alicemcneil) [inline css/portfolio.css:59]: section1 - this text looks a little small on the page

File: css/portfolio.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+/*this overides padding default*/
+
+body {
+    font-size: 62.5%;
+    font-family: "amatic sc", cursive;
+}
+/*this scales fonts when zoom is altered in OS*/
+
+nav {
+    background-color: #DCDCDC;
```
Comment (alicemcneil) [inline css/portfolio.css:13]: does this grey//white contrast stand out enough?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    <div class="image-container">
+        <div class="image-title">
+            <h1 class="title-style">title123456</h1>
+        </div>
+    </div>
+
+    <section class="section1">jhgcjghcjhg</section>
+
+    <section class="section2">
+        About me
```
Comment (alicemcneil) [inline index.html:34]: i guess 'about me' is a heading? maybe use a `<h3> ` or other heading tag here

File: css/portfolio.css [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    height: 100px;
+    padding-top: 20px;
+    color: white;
+    font-weight: bold;
+    position: fixed;
+    width: 100%;
+
+}
+.menu {
+}
```
Comment (alicemcneil) [inline css/portfolio.css:23]: `letter-spacing: 5px;` this is a way to make space between your letters, might help to make your buttons more readable even when you decrease the size

File: css/portfolio.css [hunk 6]
```diff
@@ -0,0 +1,92 @@
+* {
```
Comment (alicemcneil) [inline css/portfolio.css:1]: i really like this design and font choice! :)

File: index.html [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    <footer class="footer"> CONTACT </footer>
+
+
+
+
+</body>
+
+
+
+</html>
```
Comment (alicemcneil) [inline index.html:51]: well done, this is really good! proud of you :)

**Review by sjr-mayden — APPROVED** [review]

**Pull request merged**